packages/@aws-cdk-testing/framework-integ/test/aws-eks/test/integ.alb-controller.ts

```diff
@@ -20,7 +20,7 @@ class EksClusterAlbControllerStack extends Stack {
       vpc,
       ...getClusterVersionConfig(this),
       albController: {
-        version: eks.AlbControllerVersion.V2_4_1,
+        version: eks.AlbControllerVersion.V2_5_1,
```

Comment:
the fact that this integ test works is good enough for me. We can be unit-test exempt (or just have one unit test if you really really want).